Automattic/wp-calypso
# A4A > Partner Directory: Implement multi-select for industry field

Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/727

## Proposed Changes

This PR implements multi-select for the Partner Directory industry field.

**Note:** This PR requires this diff: D153748-code

## Testing Instructions

1. Open the A4A live link.
2. Go /partner-directory/agency-details
3. Scroll down to `Listing details` section and verify the name `Industry` is changes is `Industries` & you can add multiple values.

<img width="742" alt="Screenshot 2024-06-28 at 5 02 06 PM" src="https://github.com/Automattic/wp-calypso/assets/10586875/c1dfb795-5b2e-4cc1-9e77-8a7fd2f9ba78">


## Pre-merge Checklist
